From fa9815285c536aca6d8a1bbebbe34e9f26cbd883 Mon Sep 17 00:00:00 2001
From: Oliver Brown <oliver.brown@nxp.com>
Date: Mon, 16 Jul 2018 07:58:25 -0500
Subject: [PATCH 4323/5242] MLK-18904-1: hdp: Add memory region for iMX8M
 reset control

commit  b1cb3facba1e2ae612b94cca8d5ad95c04381679 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add reset memory region to allow reset of the HDMI/DisplayPort PHY.

Signed-off-by: Oliver Brown <oliver.brown@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/hdp/util.h |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/mxc/hdp/util.h b/drivers/mxc/hdp/util.h
index 063c369..5cf3c81 100644
--- a/drivers/mxc/hdp/util.h
+++ b/drivers/mxc/hdp/util.h
@@ -35,7 +35,7 @@
  * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE
  * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  *
- * Copyright 2017 NXP
+ * Copyright 2017-2018 NXP
  *
  ******************************************************************************
  *
@@ -181,6 +181,7 @@
 struct hdp_mem {
 	void __iomem *regs_base; /* Controller regs base */
 	void __iomem *ss_base; /* HDP Subsystem regs base */
+	void __iomem *rst_base; /* HDP Subsystem reset base */
 };
 
 struct hdp_rw_func {
-- 
1.7.9.5

